rfcs/README.md

```diff
@@ -0,0 +1,157 @@
+# RFCs at Fission
+
+This introduction should provide you everything you need to get you started in
+the wonderful world of RFCs at Fission. Read on to discover how our organization
+handles proposals for additions and changes to the p# RFCs at Fission
```

Comment:
Is `p#` a convention that I'm not familiar with?

Reply:
that's just a typo :).

Reply:
fixed in next push.